pkg/derivation/parser.go

```diff
@@ -288,5 +289,10 @@ func arrayEach(value []byte, callback func(value []byte, index int) error) error
 }
 
 func unquote(b []byte) string {
-	return string(b[1 : len(b)-1])
+	s, err := strconv.Unquote(string(b))
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Can we wrap the error and extend the signature to pass it along, rather than panic-ing?